Mathlib/GroupTheory/Perm/Cycle/Factors.lean

```diff
@@ -607,7 +607,7 @@ theorem cycle_is_cycleOf {f c : Equiv.Perm α} {a : α} (ha : a ∈ c.support)
   let hfc := (Equiv.Perm.disjoint_mul_inv_of_mem_cycleFactorsFinset hc).symm
   let hfc2 := Perm.Disjoint.commute hfc
   rw [← Equiv.Perm.cycleOf_mul_of_apply_right_eq_self hfc2]
-  simp only [hfc2.eq, inv_mul_cancel_right]
+  · simp only [hfc2.eq, inv_mul_cancel_right]
```

Comment:
I wouldn't mind the comment below this line getting translated (I speak some French, but not everybody does) - but this is pre-existing.

Reply:
Translation provided!